fileshare/event_manager.go

```diff
@@ -296,8 +304,8 @@ func (em *EventManager) EventFunc(eventJSON string) {
 		}
 		transfer, ok := em.transfers[event.TransferID]
 		if !ok {
-			log.Printf("transfer %s from requestQueued event not found", event.TransferID)
-			return
+			em.transfers[event.TransferID] = NewOutgoingTransfer(event.TransferID, event.Peer, "")
```

Comment:
[minor] wouldn't work to create first the transfer and then add it to the map, to save a search :smile:?
```
transfer = NewOutgoingTransfer(event.TransferID, event.Peer, "")
em.transfers[event.TransferID] = transfer
```